src/model/encoding/__tests__/convertFromHTMLToContentBlocks-test.js

```diff
@@ -45,4 +45,16 @@ describe('convertFromHTMLToContentBlocks', () => {
     'p',
     'pre',
   ].forEach(tag => testConvertingAdjacentHtmlElementsToContentBlocks(tag));
+
+  it('applies textDecoration with multiple styles', () => {
+    const html = '<span style="text-decoration:underline line-through;">okurrrr</span>';
+    const blocks = convertFromHTMLToContentBlocks(html);
+    const charList = blocks.contentBlocks[0].characterList;
+
+    charList.forEach(charMetadata => {
+      const { style } = charMetadata;
+      expect(style).toContain('UNDERLINE');
+      expect(style).toContain('STRIKETHROUGH');
+    });
+  });
```

Comment:
🎉 TESTS! Thank you!

Reply:
Can we also assert that there are only two values in `style`

Reply:
tests!! _yes_ts!!